src/runtime_src/core/pcie/tools/xbutil/xbutil.h

```diff
@@ -715,14 +710,13 @@ class device {
     int readSensors( void ) const
     {
         // board info
-        std::string name, vendor, device, subsystem, subvendor, xmc_ver,
+        std::string vendor, device, subsystem, subvendor, xmc_ver,
             ser_num, bmc_ver, idcode, fpga, dna, errmsg, max_power;
         int ddr_size = 0, ddr_count = 0, pcie_speed = 0, pcie_width = 0, p2p_enabled = 0;
         std::vector<std::string> clock_freqs;
         std::vector<std::string> dma_threads;
         bool mig_calibration;
         
-        pcidev::get_dev(m_idx)->sysfs_get( "rom", "VBNV",               errmsg, name ); 
         pcidev::get_dev(m_idx)->sysfs_get( "", "vendor",                errmsg, vendor );
```

Comment:
Considering cache vendor/device/subdev since they don't change?

Reply:
I am not sure how that might be helpful. I don't see vendor/device/subdev being used anywhere else